src/base/utils/string.cpp

```diff
@@ -56,7 +55,12 @@ QString Utils::String::fromDouble(const double n, const int precision)
 #if (QT_VERSION >= QT_VERSION_CHECK(6, 0, 0))
 QString Utils::String::wildcardToRegexPattern(const QString &pattern)
 {
-    return QRegularExpression::wildcardToRegularExpression(pattern, QRegularExpression::UnanchoredWildcardConversion);
+    // replace [ and ] with [[] and []]
+    // https://doc-snapshots.qt.io/qt6-dev/qregularexpression.html#wildcardToRegularExpression
```

Comment:
I would suggest to remove this comment, it isn't particularly useful IMO.

Reply:
I'd vote to keep it, as I think a comment such as:
```
// escape [ and ] characters
```
is much easier to understand than:
```
.replace(QRegularExpression(u"\\[|\\]"_qs), u"[\\0]"_qs)
```
Then, as we have to escape the brackets by using… brackets, I think it is worth explaining in the comment.

Reply:
I meant this line: `// https://doc-snapshots.qt.io/qt6-dev/qregularexpression.html#wildcardToRegularExpression` not the one above.

Reply:
Force-pushed to adjust code comments.